File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/StringPrimitiveListInPerson.java
package com.kinvey.androidTest.model;

import com.google.api.client.util.Key;

import java.util.List;

/**
 * Created by yuliya on 09/20/17.
 */

public class StringPrimitiveListInPerson extends Person {

    @Key("stringList")
    private List<String> stringList;

    public StringPrimitiveListInPerson() {
    }

    public StringPrimitiveListInPerson(String username) {
        this.username = username;
    }

    public List<String> getStringList() {
        return stringList;
    }

    public void setStringList(List<String> stringList) {
        this.stringList = stringList;
    }
}

File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/StringPrimitiveListInPerson.kt
package com.kinvey.androidTest.model

import com.google.api.client.util.Key

/**
 * Created by yuliya on 09/20/17.
 */

data class StringPrimitiveListInPerson(
    @Key("stringList")
    var stringList: List<String>? = null
): Person() {
    constructor(username: String?): this() {
        this.username = username
    }
}